### PR TITLE
doc: Release notes for NCS 1.3.1

### PR DESCRIPTION
**Pull request by umapraseeda**
Release notes for NCS 1.3.1

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

**Comment by umapraseeda** [comment]
[nRF Connect SDK v1.3.1 Release Notes — nRF Connect SDK 1.3.0 documentation.pdf](https://github.com/nrfconnect/sdk-nrf/files/5201874/nRF.Connect.SDK.v1.3.1.Release.Notes.nRF.Connect.SDK.1.3.0.documentation.pdf)

**Review by MonikaLieLarsen — APPROVED** [review]

**Review by ru-fu — APPROVED** [review]
Looks good! :)

File: doc/release-notes-1.3.1.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.1 for prototyping and evaluation.
+Support for production and deployment in end products is coming soon.
+
+
+Highlights
+**********
+
+* LwM2M carrier library v0.9.1 is certified with AT&T.
+* Bluetooth LE stack (consisting of nRF Bluetooth LE Controller and Zephyr Host) is now Bluetooth Core Specification v5.2 qualified.
```
Comment (guwa) [inline doc/release-notes-1.3.1.rst:18]: According to the link provided by @joerchan in https://projecttools.nordicsemi.no/confluence/pages/viewpage.action?pageId=78820825&focusedCommentId=78824266#comment-78824266

The Zephyr host is 5.1 qualified, and SoftDevice controller is 5.2 qualified.

Comment (joerchan) [reply doc/release-notes-1.3.1.rst:18]: That is correct.

Comment (joerchan) [inline doc/release-notes-1.3.1.rst:18]:
```suggestion
* Bluetooth Host is now Bluetooth Core Specification v5.1 qualified.
* Bluetooth LE controller (nRF Bluetooth LE Controller) is now Bluetooth Core Specification v5.2 qualified.
```

Comment (ru-fu) [reply doc/release-notes-1.3.1.rst:18]: Would that make it clearer @joerchan ?
```suggestion
* Bluetooth Host is now Bluetooth Core Specification v5.1 qualified.
* nRF Bluetooth LE Controller (in sdk-nrfxlib) is now Bluetooth Core Specification v5.2 qualified.
```
With your suggestion, I would be confused if the Zephyr Controller is qualified or not.

Comment (joerchan) [reply doc/release-notes-1.3.1.rst:18]: @ru-fu That is a good point. Let's go with yours.

Comment (umapraseeda) [reply doc/release-notes-1.3.1.rst]: Added this change.

File: doc/release-notes-1.3.1.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.1 for prototyping and evaluation.
+Support for production and deployment in end products is coming soon.
+
+
+Highlights
+**********
+
+* LwM2M carrier library v0.9.1 is certified with AT&T.
+* Bluetooth Host is now Bluetooth Core Specification v5.1 qualified.
+* nRF Bluetooth LE Controller (in sdk-nrfxlib) is now Bluetooth Core Specification v5.2 qualified.
```
Comment (shantha-14) [inline doc/release-notes-1.3.1.rst:19]: It is now renamed as softdevice_controller, should not this stated in highlights @jdstapes / @bjorn-tore-taraldsen

Comment (ru-fu) [reply doc/release-notes-1.3.1.rst:19]: It has been renamed on master, but it's still nRF Bluetooth LE Controller on the v1.3 branch.

**Review by bjorn-tore-taraldsen — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

File: doc/release-notes-1.3.1.rst (hunk 2 above)
Comment (shantha-14) [reply doc/release-notes-1.3.1.rst:19]: OK

**Pull request merged**